message_sender/views.py

```diff
@@ -137,13 +138,17 @@ def post(self, request, *args, **kwargs):
                                 request.data["nack_reason"]
                             message.save()
                         send_message.delay(str(message.id))
-
                         if "voice_speech_url" in message.metadata:
                             fire_metric.apply_async(kwargs={
                                 "metric_name":
                                     'vumimessage.obd.unsuccessful.sum',
                                 "metric_value": 1.0
                             })
+                    outbound_type = "voice" if "voice_speech_url" in \
+                        message.metadata else "text"
+                    ConcurrencyLimiter.decr_message_count(
+                        outbound_type, message.last_sent_time)
```

Comment:
I've moved this code in 583f570 but I'm pretty confused about events vs. inbounds so I'm really not sure if I've done the right thing.